utils/docker/Dockerfile.centos.7

```diff
@@ -210,7 +210,7 @@ RUN if [ "x$NOBUILD" = "x" ] ; then \
 # Build java and hadoop bindings
 RUN if [ "x$NOBUILD" = "x" ] ; then \
     pushd src/client/java && \
-    mvn clean install -T 1C -DskipITs -Ddaos.install.path=/opt/daos && \
+    mvn clean install -T 1C -DskipITs -Dgpg.skip -Ddaos.install.path=/opt/daos && \
```

Comment:
If have added a skip`-Dgpg.skip` for java build process, this should be updated in the README as well https://github.com/daos-stack/daos/blame/master/src/client/java/README.md#L123

Reply:
thanks for catching that. I'll add it in the README.md later.